ducktape/cluster/remoteaccount.py

```diff
@@ -62,6 +63,8 @@ def _can_ping_url(self, url, headers):
     def ssh_command(self, cmd):
         if self.local:
             return cmd
+        if self.sudo_user is not None:
```

Comment:
I agree with can allow user to specify `sudo_user` per ssh_command to play on the safe side, just in case user will need that. It's just that I don't have a scenario that user will need that yet.